workflows/rules/qc/insilico_coverage.smk

```diff
@@ -0,0 +1,33 @@
+# vim: syntax=python tabstop=4 expandtab
+# coding: utf-8
+
+import os
+
+def get_bedfile_path(wcs):
+    return config["insilico"][f"{wcs.insiliconame}"]['bedfile']
+
+def get_bedfile_version(wcs):
+    return config["insilico"][f"{wcs.insiliconame}"]['version']
+
+rule insilico_coverage:
+    input:
+        bamfile = "{workingdir}/dedup/{sampleid}_DEDUP.bam"
```

Comment:
the files are under {workingdir}/{stype}/dedup/{sname}_DEDUP.bam
(stype = tumor or normal)